### PR TITLE
feat: finetuning data generation

### PR DESCRIPTION
**Pull request by jupyterjazz**
fine-tuning data generation for dev branch

### CODE REVIEW

File: tests/resources/nest/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wordlift-multi-modal-search",
-  "extracted_dataset": "extracted-wordlift-data",
+  "extracted_dataset": "wordlift-test-data-1",
```
Comment (violenil) [inline tests/resources/nest/config.json:3]:
```suggestion
  "dataset_name": "extracted-data-wordlift-50-flat",
```

**Review by violenil — APPROVED** [review]
🚀

File: tests/resources/nest/config.json (hunk 1 above)
Comment (violenil) [reply tests/resources/nest/config.json:3]: Note: we should also read our `UserInfo.dataset` from this config field

Comment (jupyterjazz) [reply tests/resources/nest/config.json:3]: I'll change this after I merge to dev branch (and have data loading functionality) otherwise tests will fail

**Pull request merged**